runtime/bcverify/bcverify.c

```diff
@@ -2727,7 +2727,7 @@ j9bcv_J9VMDllMain (J9JavaVM* vm, IDATA stage, void* reserved)
 							returnVal = J9VMDLLMAIN_FAILED;
 						}
 					} else {
-						loadInfo->fatalErrorStr = "No options specified for -Xverify:<opt>";
```

Comment:
Should we initialize `errorString` to `loadInfo->fatalErrorStr`? Since within `setVerifyState` (which gets called from `parseOptions`) we only update `errorString` if it is non-`NULL`. Would initializing `errorString` to `NULL` put us in danger of changing the behaviour of `parseOptions` and `setVerifyState`?

Reply:
No, I don't think that would be appropriate initialize `errorString` that way.

We're not passing `NULL` to `parseOptions()` (nor to `setVerifyState()`), we're passing `&errorString` which is the address of that local. Only if `parseOptions()` fails (by returning `FALSE`) do we want to call `setErrorJ9dll()`.